html/partials/50x.shtml

```diff
@@ -1,2 +1,3 @@
-<p>Sorry, the page you are looking for is currently unavailable.</p>
-<p>Please try again later.</p>
+<h2>An error occurred</h2>
+<p>The page you’re looking for isn’t available.</p>
```

Comment:
Or we could go with "There's a problem with this service." like on the 500 page?..